### DISCUSSION TITLE
Question about the future meteorological data

### DISCUSSION THREAD

**Discussion by moonquakes66** [PaddlePaddle/PaddleSpatial#46] [BaiduKDDCup2022] [answered]
Do the future meteorological data include these columns like Ndir or Pab1(and Pab2,Pab3)?What is the whole of  future meteorological data?

**Comment by zhoujingbo** [comment, marked as answer]
There is no future meteorological data for prediction.

For the meteorological data question, please refer to: https://github.com/PaddlePaddle/PaddleSpatial/discussions/16

**Reply by moonquakes66** [reply]
Thanks. And are the yaw angle of the nacelle or pitch angle of blade included in  future meteorological data? I don't know the exact conception of  future meteorological data.

**Reply by zhoujingbo** [reply]
The yaw angle of the nacelle or pitch angle of blade will  not provide for test prediction
